manifests/params.pp

```diff
@@ -147,13 +147,17 @@
   $allow_weak_random_seed          = true
 
   $ssl_version                 = 'TLSv1.2+'
+
+  # adh is required with cipher list below
+  # see: https://github.com/NagiosEnterprises/nrpe/issues/119
+  $ssl_use_adh                 = 2
   $ssl_ciphers                 = [
     'DHE-RSA-AES128-GCM-SHA256',
     'DHE-RSA-AES256-GCM-SHA384',
     'DHE-RSA-AES128-SHA',
     'DHE-RSA-AES256-SHA',
     'DHE-RSA-AES128-SHA256',
-    'DHE-RSA-AES256-SHA256',
+    'DHE-RSA-AES256-SHA256'
```

Comment:
This does not look right. I would expect CI to catch this (we want trailing comas)… weird… can you revert it?

Reply:
I will fix this.